v2/internal/frontend/desktop/darwin/WailsContext.h

```diff
@@ -66,7 +66,11 @@
 - (void) UnFullscreen;
 - (void) Minimise;
 - (void) UnMinimise;
+- (bool) IsMinimized;
```

Comment:
We should rename this `IsMinimised` to keep the naming convention consistant.

Reply:
Is renamed